code/modules/cooking/plates.dm

```diff
@@ -0,0 +1,182 @@
+/*
+Plates that can hold your cooking stuff
+*/
+//Click with food to put on plate
+//Click with cutlery to take some
+//Click in hand to remove food
+
+/obj/item/reagent_containers/bowl
+	name = "bowl"
+	desc = "A small bowl for serving liquid meals in."
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "bowl"
+	fragile = 3
+	shatter_material = DEFAULT_TABLE_MATERIAL // Slight typecasting abuse here, gets converted to a material in Initialize().
+	can_be_placed_into = list()
+	flags = OPENCONTAINER
+	var/grease = FALSE
```

Comment:
is this set to FALSE when put in a utility vendor?

Reply:
Yes